Sources/Options.swift

```diff
@@ -806,7 +816,7 @@ public struct FormatOptions: CustomStringConvertible {
                 customTypeMarks: Set<String> = [],
                 blankLineAfterSubgroups: Bool = true,
                 alphabeticallySortedDeclarationPatterns: Set<String> = [],
-                alphabetizeSwiftUIPropertyTypes: Bool = false,
+                swiftUIPropertiesSortMode: SwiftUIPropertiesSortMode = .none,
```

Comment:
If you can figure out the `OptionDescriptor` for this, using an optional `SwiftUIPropertiesSortMode` seems maybe better than having a `none` case. If it complicates the `OptionDescriptor` then this is fine too!

```suggestion
                swiftUIPropertiesSortMode: SwiftUIPropertiesSortMode? = nil,
```

Reply:
That was my first approach but there are no `OptionDescriptors` inits that allows for an optional type. The inits require a type that conforms to `RawRepresentable`.

Reply:
What if we added an extension like `extension Optional: RawRepresentable where Wrapped: RawRepresentable, `RawValue == String`?

Reply:
I had to hardcode a `none` string but it does work 👍

Reply:
This was an ingenious approach, however conforming a type you don't own (Optional) to a protocol you don't own (RawRepresentable) generates a warning now in Xcode. I'm going to revert to the original solution of having a `none` case.